corehq/apps/users/tests/test_views.py

```diff
@@ -201,3 +209,79 @@ def test_delete_phone_number(self):
         self.assertEqual(user_history_log.by_domain, self.domain)
         self.assertEqual(user_history_log.for_domain, self.domain)
         self.assertEqual(user_history_log.changed_via, USER_CHANGE_VIA_WEB)
+
+
+class TestCountWebUsers(TestCase):
+
+    view = 'count_web_users'
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.domain = 'test'
+        cls.domain_obj = create_domain(cls.domain)
+
+        set_toggle(FILTERED_BULK_USER_DOWNLOAD.slug, cls.domain, True, namespace='domain')
```

Comment:
FYI you can alternatively decorate the test with `@flag_enabled('FILTERED_BULK_USER_DOWNLOAD')`